commons/src/main/java/org/infinispan/commons/event/KVPEvent.java

```diff
@@ -0,0 +1,30 @@
+package org.infinispan.commons.event;
+
+import java.io.Serializable;
+
+public class KVPEvent<K, V> implements Serializable {
```

Comment:
Prob makes sense to implement `toString`? `equals` and `hashCode` not so important.